packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -35,17 +35,17 @@ After you create your service mesh, you can create virtual services, virtual nod
 The following example creates the `AppMesh` service mesh with the default egress filter of `DROP_ALL`. See [the AWS CloudFormation `EgressFilter` resource](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-mesh-egressfilter.html) for more info on egress filters.
 
 ```ts
-const mesh = new Mesh(stack, 'AppMesh', {
-  meshName: 'myAwsmMesh',
+const mesh = new appmesh.Mesh(stack, 'AppMesh', {
```

Comment:
Remove the `appmesh` qualifier here and everywhere in the README. Types from the documented module should be unqualified. To make that compile, add a default fixture in this module that imports all the appmesh types that are needed. 

Details on Rosetta fixtures: https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#rosetta 

An example module-level default fixture: https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-kinesisfirehose/rosetta/default.ts-fixture

Reply:
Thanks for attaching a link to the relevant to document!